README.md

```diff
@@ -138,12 +145,109 @@ make test-ts-binary-rococo
+GPLv3
 
-![image](https://user-images.githubusercontent.com/7630809/135689832-1f57cd5c-7f83-4fce-9bb0-832b77a38dcc.png)
 
-## License
+## How to Build and Run Parachain and Tee-worker
 
-GPLv3
+## Preparation
+
+1. env: [Setup **SGX TEE** Environment](https://web3builders.notion.site/Setup-SGX-TEE-Environment-68066770831b45b7b632e682cf159477?pvs=4) 
+2. add VSCode configs for indexing and parsing tee-worker codes
+
+```
+"rust-analyzer.linkedProjects": [
+        "tee-worker/Cargo.toml"
+    ],
+```
+
+## Build
+
+We need to build the `integritee-service` before we launch the entire system
+
+```
+git clone https://github.com/litentry/litentry-parachain
+cd ./litentry-parachain/tee-worker
+source /opt/intel/sgxsdk/environment
+SGX_MODE=SW make
+```
+
+## Launch
+
+### 1. Start a local docker setup
+
+In order to create a local docker setup, you can run the following command
+```
+./local-setup/launch.py local-setup/github-action-config-one-worker.json
+```
+This will create three docker containers, 2 Relay Chain Validators and 1 Parachain Collator. However it will be using the default ports as present in .env.dev . If you want to run the system by offseting the default ports by a certain value. You can run this command instead:
+
+```
+./local-setup/launch.py local-setup/development-worker.json local-docker 100
+```
+This will run the same containers and use the offset value of 100.
+
+### 2. Start a local binary setup
+
+In order to create a local binary setup, using default ports, you can run the following command:
+```
+./local-setup/launch.py local-setup/github-action-config-one-worker.json local-binary
+```
+
+If you want to launch the same system by offseting the port values, you can use this command 
+```
+./local-setup/launch.py local-setup/development-worker.json local-binary 100
+```
+In case you receive the following error:
+```ModuleNotFoundError: No module named 'pycurl'```
+
+Fix it manually by installing pycurl using pip3. 
+
+
+### 3. Remote <> Integritee Node
+
+If you are running the integritee node manually, Then be mindful of the port that is being set while launching the integritee node, For example:
+```
+cargo build --release --features "skip-extrinsic-filtering skip-ias-check skip-scheduled-enclave-check"
+./target/release/integritee-node --rpc-cors=all --ws-external --tmp --dev  --state-pruning archive  --blocks-pruning archive --rpc-port 9933 --ws-port 9944
+```
+
+`--rpc-port` is used to set the rpc port and `--ws-port` is used to set the ws port for the node. You then have to update the `config.json` with the port that is being used and then run the command 
+
+**Note: The `Integritee-Node` is not maintained by Parachain Team actively, Please check the repository if it has all the latest updates from `Lintentry-Parachain` .**
+
+### TEE Worker Tests 
+```
+cd ./litentry-parachain/tee-worker/ts-tests
+nvm use
+corepack yarn install
+corepack yarn run test-identity:local # example
```

Comment:
The changes have been made here. Do you think it's possible to write this as: "Refer to [tee-worker ts-tests](https://github.com/litentry/litentry-parachain/blob/dev/tee-worker/ts-tests/README.md)"?

Reply:
Updated